keyboards/fluorite/readme.md

```diff
@@ -0,0 +1,15 @@
+# fluorite
+
+![fluorite](https://github.com/ihotsuno/keyboard/blob/master/kbd-fluorite-cxl/doc/image/fluorite-CXL-001.jpg)
+
+Fluorite is a chaotic keyboard consisting of staggered rows, staggered columns, and ortholinear.
+
+* Keyboard Maintainer: [mahuyu ihotsuno](https://github.com/ihotsuno)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: links to where you can find this hardware
```

Comment:
Do you have a link for this?  

A link to a store, or thread for group buy?

Reply:
link added